docs/faq.md

```diff
@@ -18,4 +18,18 @@ We also compared the mean gene expression reported for each gene by both methods
 
 Recent reports from others support our findings. 
 [He _et al._ (2021)](https://doi.org/10.1101/2021.06.29.450377)) demonstrated that Alevin-fry can process single-cell and single-nuclei data more quickly and efficiently then other available methods, while also decreasing the false positive rate of gene detection that is commonly seen in methods that utilize transcriptome alignment.
-[You _et al._ (2021)](https://doi.org/10.1101/2021.06.17.448895) and [Tian _et al._ (2019)](https://doi.org/10.1038/s41592-019-0425-8) have also noted that results from different pre-processing workflows for single-cell RNA-sequencing analysis tend to result in compatible results downstream.
\ No newline at end of file
+[You _et al._ (2021)](https://doi.org/10.1101/2021.06.17.448895) and [Tian _et al._ (2019)](https://doi.org/10.1038/s41592-019-0425-8) have also noted that results from different pre-processing workflows for single-cell RNA-sequencing analysis tend to result in compatible results downstream.
+
+#### How do I use the provided RDS files in R? 
```

Comment:
A note here but also applies elswhere... Which level of heading do we want here? Is there a reason not to stick with just H2 (`##`)?

Reply:
Just seeing this after requesting your re-review. Let me put all the headers to be H2 and then it will be ready for re-review.